frontend/src/console/components/CoursesListingItem/CoursesListingItem.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import './styles.scss';
+import { Col, Row } from 'react-bootstrap';
+import { ThemeSchema } from 'ocim-client';
+import courseImage from 'assets/course_image.jpg';
+import messages from './displayMessages';
+import { WrappedMessage } from '../../../utils/intl';
+import { CustomizableLink } from '../CustomizableLink';
+
+interface CoursesListingItemProps {
+  themeData: ThemeSchema;
+  loggedIn?: boolean;
+}
+
+export const CoursesListingItem: React.FC<CoursesListingItemProps> = (
+  props: CoursesListingItemProps
+) => {
+  const { themeData } = props;
+  const textStyle = { color: themeData.mainColor };
+  // TODO: Make text customizable.
+
+  return (
+    <Row
+      className="courses-listing-item"
+      style={{ borderBottomColor: themeData.linkColor }}
```

Comment:
@giovannicimolin, should this be altered as well? I see alternative color for the bottom border of the `courses listing item` on the mockups, but I'm not sure if it's related to `linkColor` variable.

Reply:
@Agrendalath The thing is that we're not 100% sure how the generic variables (`main` and `link` colors) are applied to the instances, so we should schedule a follow up and fix any issues when assigning color.

For this ticket, I'm happy to merge as the preview component was implemented.